editor/asset_library_editor_plugin.cpp

```diff
@@ -808,6 +808,7 @@ void EditorAssetLibrary::_request_image(ObjectID p_for, String p_image_url, Imag
 	iq.image_index = p_image_index;
 	iq.image_type = p_type;
 	iq.request = memnew(HTTPRequest);
+	id.asset_id = 0;
```

Comment:
That's invalid: https://travis-ci.org/godotengine/godot/builds/259901322?utm_source=github_status&utm_medium=notification